src/components/scenes/RequestScene.tsx

```diff
@@ -82,7 +88,7 @@ interface AddressInfo {
 
 const inputAccessoryViewID: string = 'cancelHeaderId'
 
-export class RequestSceneComponent extends React.Component<Props, State> {
+export class RequestSceneComponent extends React.Component<Props & HookProps, State> {
```

Comment:
Conjoin HookProps in the definition of `Props`.

Reply:
Because `RequestSceneConnected` specifically Receives a combo of just `OwnProps` and `HookProps`. Only the `RequestSceneComponent` gets everything. I needed to bifurcate the prop types.